### PR TITLE
Feature/pi 1998 mas sentence tab previous orders

### PR DESCRIPTION
**Pull request by achimber-moj**

### CODE REVIEW

File: server/views/pages/sentence.njk [hunk 1]
```diff
@@ -239,10 +239,10 @@
     {% set history %}
         {% set previousOrders %}
             {% if sentenceDetails.probationHistory.numberOfTerminatedEvents == 1 %}
-              <a href="/case/{{ crn }}/sentence/previous-orders">1 previous order</a><br />
+              <a href="/case/sentence/{{ crn }}/previous-orders">1 previous order</a><br />
```
Comment (pmcphee77) [inline server/views/pages/sentence.njk:242]: I think the route should still be `/case/{{ crn }}/sentence/previous-orders`

**Review by pmcphee77 — CHANGES_REQUESTED** [review]

File: server/data/model/previousOrderHistory.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line import/no-cycle
+
+import { Name } from './common'
+
+export interface previousOrderHistory {
+  name: Name
+  previousOrders: previousOrder[]
+}
+
+export interface previousOrder {
```
[the remark this answers is not in the document]
Comment (achimber-moj) [reply server/data/model/previousOrderHistory.ts]: done

File: server/data/model/previousOrderHistory.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line import/no-cycle
+
+import { Name } from './common'
+
+export interface previousOrderHistory {
```
[the remark this answers is not in the document]
Comment (achimber-moj) [reply server/data/model/previousOrderHistory.ts]: done

File: server/views/pages/sentence.njk (hunk 1 above)
Comment (achimber-moj) [reply server/views/pages/sentence.njk:242]: b/e api matches /case/sentence/{{ crn }}/previous-orders.  can we pick this up on a later refactor

Comment (achimber-moj) [reply server/views/pages/sentence.njk]: f/e change done.  will most likely leave the b/e

**Review by pmcphee77 — CHANGES_REQUESTED** [review]

**Review by pmcphee77 — APPROVED** [review]

**Pull request merged**